bitnami/argo-workflows/values.yaml

```diff
@@ -0,0 +1,1075 @@
+  ##
+  customReadinessProbe: {}
+  ## controller resource requests and limits
+  ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+  ## @param controller.resources.limits The resources limits for the controller containers
+  ## @param controller.resources.requests The requested resources for the controller containers
+  ##
+  resources:
+    limits: {}
+    requests: {}
+  ## Configure Pods Security Context
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod
+  ## @param controller.podSecurityContext.enabled Enabled controller pods' Security Context
+  ## @param controller.podSecurityContext.fsGroup Set controller pod's Security Context fsGroup
+  ##
+  podSecurityContext:
+    enabled: true
+    fsGroup: 1001
+  ## Configure Container Security Context
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod
+  ## @param controller.containerSecurityContext.enabled Enabled controller containers' Security Context
+  ## @param controller.containerSecurityContext.runAsUser Set controller containers' Security Context runAsUser
+  ##
+  containerSecurityContext:
+    enabled: true
+    runAsUser: 1001
+
+  ## Server container port
+  ## @param controller.containerPorts.controller Argo Workflows controller container port
+  containerPorts:
+    controller: 6060
+
+  ## Create RBAC resources for the Argo workflows controller
+  ## @param controller.rbac.create Create RBAC resources for the Argo workflows controller
+  ##
+  rbac:
+    create: true
+
+  ## Use an existing configmap instead of creating a configmap with the specified parameters
+  ## @param controller.existingConfigMap
+  existingConfigMap:
+
+  ## @param controller.extraArgs Extra arguments for the controller command line
+  ##
+  extraArgs: ""
+
+  ## Controller configmap configuration content. Requires controller.existingConfigmap to be empty.
+  ## @param controller.config [object] Controller configmap configuration content
+  config: |
+    {{- if .Values.controller.instanceID.enabled }}
+    {{- if .Values.controller.instanceID.useReleaseName }}
+    instanceID: {{ .Release.Name }}
+    {{- else }}
+    instanceID: {{ .Values.controller.instanceID.explicitID }}
+    {{- end }}
+    {{- end }}
+    ## Ref: https://argoproj.github.io/argo-workflows/workflow-executors/
+    containerRuntimeExecutor: {{ .Values.controller.containerRuntimeExecutor }}
+    ## Uncomment to create containerRuntimeExecutors
+    ## Ref: https://argoproj.github.io/argo-workflows/workflow-executors/
+    ## containerRuntimeExecutors: {}
+    ## How many workflows can be running at the same time
+    parallelism:
+    ## Maximun number of workflows running in a namespace
+    namespaceParallelism:
+    {{- if or .Values.executor.resources .Values.executor.extraEnvVars .Values.executor.podSecurityContext }}
+    executor:
+      {{- if .Values.executor.resources }}
```

Comment:
Can you set the executor imagePolicy here? Not sure if these are elements added to the executor container.

Reply:
added to the CLI